doc/debugging.rst

```diff
@@ -78,4 +78,3 @@ References
 
 * `ROS 2 and GDB <https://juraph.com/miscellaneous/ros2_and_gdb/>`_
 * `Using GDB to debug a plugin <https://stackoverflow.com/questions/10919832/how-to-use-gdb-to-debug-a-plugin>`_
```

Comment:
```suggestion
* `Using GDB to debug a plugin <https://stackoverflow.com/questions/10919832/how-to-use-gdb-to-debug-a-plugin>`_
* `GDB CLI Tutorial <https://www.cs.umd.edu/~nelson/classes/resources/cdebugging/gdb/gdb-reference-card.pdf>`_
```

Reply:
```suggestion
* `Using GDB to debug a plugin <https://stackoverflow.com/questions/10919832/how-to-use-gdb-to-debug-a-plugin>`_
* `GDB CLI Tutorial <https://users.ece.utexas.edu/~adnan/gdb-refcard.pdf>`_
```

Maybe, we can use one of these two.

Do you think it makes sense to host these files locally in github and cross-reference them?

Reply:
I don't know what was the content of the old link, the new ones are no "tutorial"

Reply:
I remember it being like a cheatsheet. May be I'm wrong. Let's see what @firesurfer @fmauch says